apps/zotonic_mod_development/src/mod_development.erl

```diff
@@ -250,3 +336,27 @@ observe_admin_menu(#admin_menu{}, Acc, Context) ->
 
      |Acc].
 
+exec_chrome(Executable, SiteUrl, ExtraArgs, #{secure := true} = Options) ->
+    TmpPath = filename:join([<<"/">>, <<"tmp">>, <<"foo">>]),
```

Comment:
To fetch the tmp dir you can use `z_tempfile:temppath()` which also checks the `TMPDIR` and `TEMPDIR` env variables.

Then you would get something like:

```erlang
TmpPath = filename:join([ z_tempfile:temppath(), "zotonic-chrome" ]),
```